packages/react/spec/imports/esm-auto-import.mjs

```diff
@@ -1,2 +1,4 @@
 import { AutoForm as PolarisAutoForm } from "@gadgetinc/react/auto/polaris";
-import { AutoForm as MUIAutoForm } from "@gadgetinc/react/auto/mui";
+// TODO: Uncomment this line when MUI autocompoents are ready for pre-release
```

Comment:
Is this the best way to temporarily remove the MUI components?
If so, should we create an Autocomponents ticket with detailed instructions on how undo the work done in this PR?

Reply:
I think these lines should be removed altogether rather than left in a commented state.

Reply:
We can refer back to this PR when it comes time to reenable them, so a linear ticket with that seems good.

Reply:
The comments are removed, and the ticket has been created: GGT-6795